src/pages/reset/reset.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <div class="container">
+    <a-form
+      ref="resetPasswordForm"
+      :rules="rules"
+      :model="form"
+      @submit.prevent
+    >
+      <h1 class="title">
+        Reset Password
+      </h1>
+
+      <p
+        v-if="formErrors.token"
+        class="common-error"
+      >
+        {{ formErrors.token }}
+      </p>
+
+      <p>Please choose your new password</p>
+
+      <a-form-item name="password">
+        <a-input
+          v-model:value="form.password"
+          type="password"
+          placeholder="Password"
+          autocomplete="off"
+        />
+      </a-form-item>
+
+      <a-form-item>
+        <a-button
+          type="primary"
+          html-type="submit"
+          :disabled="isLoading"
+          @click="handleSubmit"
```

Comment:
maybe should add `handleSubmit` to `<a-form>` element and remove from here?

Reply:
I don't like submit form on Enter :)

Reply:
@elena-vlasova 
Why? Submit on enter is common for any form and also accessibility-friendly
https://stackoverflow.com/questions/57064691/is-implicit-submission-submit-on-enter-key-useful-to-people-using-readers-and

Reply:
Fixed